app/views/devise/mailer/confirmation_instructions.html.slim

```diff
@@ -11,13 +11,16 @@ table.button.expanded.large.radius
               td
                 center
                   = link_to t('mailer.confirmation_instructions.link_text'), \
-                    sign_up_create_email_confirmation_url(confirmation_token: @token), \
+                    sign_up_create_email_confirmation_url(_request_id: \
```

Comment:
Yeah I think that is ideal, but that cannot always be depended on. I will noodle on ways we can make this more obvious, perhaps through some specs